llvm/test/CodeGen/AMDGPU/bundle-latency.mir

```diff
@@ -14,7 +14,7 @@ body:             |
     ; GCN-NEXT:   $vgpr1 = GLOBAL_LOAD_DWORD undef $vgpr3_vgpr4, 4, 0, implicit $exec
     ; GCN-NEXT: }
     ; GCN-NEXT: $vgpr6 = V_ADD_F32_e32 killed $vgpr0, $vgpr0, implicit $mode, implicit $exec
-    ; GCN-NEXT: $vgpr5 = V_ADD_F32_e32 killed $vgpr1, $vgpr1, implicit $mode, implicit $exec
+    ; GCN-NEXT: $vgpr5 = V_ADD_F32_e32 killed $vgpr1, $vgpr1, implicit killed $mode, implicit killed $exec
```

Comment:
This is adding kills of reserved registers, which may be harmless but why does it happen? Is it a known difference between `LivePhysRegs::available` and `LiveRegUnits::available`?

Reply:
Reserved registers are not considered in LiveRegUnits. LivePhysRegs considers those.